addons/binding/org.openhab.binding.airquality/src/main/java/org/openhab/binding/airquality/internal/AirQualityConfiguration.java

```diff
@@ -0,0 +1,23 @@
+/**
+ * Copyright (c) 2014-2016 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.airquality.internal;
+
+/**
+ * The {@link AirQualityConfiguration} is the class used to match the
+ * thing configuration.
+ *
+ * @author Kuba Wolanin - Initial contribution
+ */
+public class AirQualityConfiguration {
```

Comment:
Use private members and public getter methods on the Configuration object please. This construct is slightly surprising for other Java programmers 😈